rosetta/jax_gen_pb.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
Could you please provide some documentation for this script, e.g. what it does, what arguments/options it takes, and what it produces?